examples/ai-billing/app/keys/client.tsx

```diff
@@ -0,0 +1,91 @@
+"use client";
+import {
+  Card,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeaderCell,
+  TableRow,
+} from "@tremor/react";
+import { Button } from "@tremor/react";
+import { Minus } from "lucide-react";
+import { Plus } from "lucide-react";
+import { RevokeKeyButton } from "./revoke-key";
+
+import { useState } from "react";
+import { toast } from "sonner";
+import { createKey } from "./action";
+
+export default function Client({ keys }: { keys: any[] }) {
+  const [isLoading, setLoading] = useState(false);
+  const [newKey, setNewKey] = useState<string | undefined>(undefined);
+
+  const curl = newKey
+    ? `curl --location 'http:/localhost:3000/api/openai' \\
```

Comment:
This should use the prod url when deployed, right?

Reply:
Removed this unused file